### PR TITLE
My portfolio heading pull request

### PR DESCRIPTION
**Pull request by fabriceMUKARAGE**
all my code is in the heading branch, which is both HTML and css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/*  my html code start from here */
```
Comment (brainconnect93) [inline index.html]: - [ ] Kindly remove this piece of text from your work as it is currently showing on the UI and it is not needed. 👍

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @fabriceMUKARAGE, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [ ] I can't highlight the line of code from your work to make an inline comment that is why i am dropping the request here:
    - On line `26` of your HTML code, kindly wrap your logo inside of an `anchor tag`.
      > The logo (if the design has one) is on the left and has a link (`a` tag). This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.
    E.G: 
    ```HTML
      <div class="logo"><a href="#">My logo</a></div>
    ```

   - On your README file, The MIT license link is currently broken. Kindly add the LICENSE file to your work as this gives the visitor to have access to use your work. And be sure the file is properly targeted in your README file in order for the link to work perfectly.


N.B: Always be sure you have all your work in the branch you want to submit for pull request as this will make it easier for a CR to navigate through all the changes you have in your work. Currently in your work I can only see your HTML file  in the pull request you submitted which implies that you were working on the master branch before you switch to the new header branch. For future purposes always have all your changes in the branch you are submitting for pull request.

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (fabriceMUKARAGE) [reply index.html]: @brainconnect93  Noted with thanks, for the next project, I am going to improve on that.

Comment (fabriceMUKARAGE) [reply index.html]: @brainconnect93 Noted all the feedback. Thank you

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @fabriceMUKARAGE , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] As requested by the previous reviewer, kindly make your logo a link by wrapping it in `a` tag. You can follow the example of the previous reviewer.

## Optional suggestions

- [ ] From onwards, kindly give a meaningful description to your PR.
- [ ] Kindly provide a bit more margin between the paragraph and **LET'S CONNECT**  heading as per the Figma template your are following. ⬇️
image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fabriceMUKARAGE** [comment]
@muneebulrehman Thank you for the feedback, Please I have done all the required feedback to change. Thanks

File: index.html [hunk 2]
```diff
@@ -22,7 +22,7 @@
       </ul>
     </div>
     <header>
-      <div class="logo">My logo</div>
+      <a class="logo">My logo</a>
       <nav>
```
Comment (TadesseDev) [inline index.html:26]: - [Optional] Good job making your logo a link, But as you can see the font size is very small and not as visible on the figma design. Please consider updating the font size or wrapping your link inside a `h2` tag, to make it stand out. 

#### Something like. 
```html
<h2><a class="logo">My logo</a></h2>
```
- [Optinal] Also, consider adding some spacing on the top, This Is to align your design better with the figma design. 
Screenshot 2023-02-15 102257

**Review by TadesseDev — APPROVED** [review]
### Hi @fabriceMUKARAGE 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fabriceMUKARAGE** [comment]
@Tadesse-Alemayehu Thank you very much for the insightful feedback

**Pull request merged**